### ISSUE TITLE
Intent of "and" and "or" keywords in mapping file

### ISSUE THREAD

**Issue by jhpoelen** [jhpoelen/fb-osmose-bridge#76]
@agruss2 I noticed that terms "or" and "and" are used in the mapping file.

Could you please provide examples of what you'd like to accomplish with this? 

For instance:

mapping to from "RelFecundityMean and SpawningCycles"	to "species.relativefecundity.sp" means that if the first is not available, the value of the second (i.e. SpawningCycles) should be used.

**Comment by agruss2** [comment]
@jhpoelen 
"and" means that several FishBase columns have to be used to generate a parameter value. Here, we are talking about "composite parameters", which are not the focus of API efforts currently. 
The meaning of "and" is actually clear if you have a look at the columns entitled "Notes from Arnaud".

"or" appears in only one row, for the "predation.accessibility.stage.threshold.sp" parameter. Looking at the column "Notes from Arnaud", one can read: 
The "predation.accessibility.stage.threshold.sp" parameter is the age threshold that separates the accessibility coefficients of juvenile and adult stages of focal functional groups. If predation.accessibility.stage.structure = age then the bridge between FishBase/SeaLifeBase should provide an estimate of age at sexual maturity. If predation.accessibility.stage.structure = size then the bridge between FishBase/SeaLifeBase should provide an estimate of size at sexual maturity.  
In other words, if the value of the "predation.accessibility.stage.structure" parameter is "age", then the FishBase column that should be used is "tm". 
And, if the value of the "predation.accessibility.stage.structure" parameter is "size", then the FishBase column that should be used is "Lm".

**Comment by agruss2** [comment]
@jhpoelen 
Thanks a lot for raising this up. I believe that we are now clear on this issue, which can be closed. Please feel free the reopen the current issue if you feel otherwise.

**Comment by jhpoelen** [comment]
@agruss2 thanks for your patience.
